src/App.tsx

```diff
@@ -1,9 +1,18 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const visibleMovies = moviesFromServer.filter(movie => {
+    const { title, description } = movie;
+    const filterTest = title.toLowerCase().includes(query.trim().toLowerCase())
+      || description.toLowerCase().includes(query.trim().toLowerCase());
+
+    return filterTest;
+  });
```

Comment:
```suggestion
  const visibleMovies = moviesFromServer.filter({ title, description }=> {
    return title.toLowerCase().includes(query.trim().toLowerCase())
      || description.toLowerCase().includes(query.trim().toLowerCase());
  });
```

Reply:
it is better to move logic of lowerCasing and includes check to separate function